test/lint-api.test.js

```diff
@@ -372,7 +372,7 @@ describe("Lint with API", () => {
         assert(linter.status === 1, `Linter status is 1 (${linter.status} returned)`);
         assert(linter.outputString.includes("warning"), "Output string contains warning");
         assert(Object.keys(linter.lintResult.files).length === 12, `Expected 2 files got ${Object.keys(linter.lintResult.files).length}`);
-        assert(linter.lintResult.summary.totalFoundErrorNumber === 19, `Expected 19 errors to ${linter.lintResult.summary.totalFoundErrorNumber}`);
+        assert(linter.lintResult.summary.totalFoundErrorNumber === 12, `Expected 12 errors to ${linter.lintResult.summary.totalFoundErrorNumber}`);
```

Comment:
Checking the console output during these tests I am confirm that we are still getting valid errors from the output. 

Just less since I am actually handling the parse errors in codenarc-factory.js